nucypher/policy/conditions/offchain.py

```diff
@@ -148,6 +148,20 @@ def query_response(self, data: Any) -> Any:
 
         return result
 
+    @staticmethod
+    def _process_result_for_eval(result: Any):
+        # strings that are not already quoted will cause a problem for literal_eval
+        if isinstance(result, str):
+            # check if already quoted; if not, quote it
+            if not (
+                (result.startswith("'") and result.endswith("'"))
+                or (result.startswith('"') and result.endswith('"'))
+            ):
```

Comment:
There's some edge cases here. e.g., `'` or `"` would both be valid quoted strings. 

Regarding edge cases I don't know how we should treat examples like `ab"c` or, being more realistic, `O'Sullivan`. The latter, for example, would be quoted as `'O'Sullivan'`, which I figure will break literal eval, unless it is double quoted, instead of single quoted.

Perhaps, the best option would be to disallow quotes to some degree, but unsure how.

Reply:
✔️ Nice catch! Made an update to properly handle quotations, and improved the testing. That being said, if there are combinations of both types on quotations in the string, I'm not sure how to handle that - that type of situation feels like an extreme edge-case which I'm ok with saying we don't support.